src/database/migrations/20241205082532-add_reports_table.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('reports', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
+			code: {
+				allowNull: false,
```

Comment:
@sumanvpacewisdom set code as unique

Reply:
@sumanvpacewisdom set this is a primary and refer this in other tables like report queries as well